src/components/AssetCard.vue

```diff
@@ -124,4 +124,8 @@ export default {
   .hide-card {
     visibility: 'hidden'
   }
+
+  .asset-title {
+    height: 2em;
```

Comment:
rem might be better here. Added `2.0rem` works but not tested with all browsers so adding a bit extra:
```suggestion
    height: 2.1rem;
```